example/src/App.tsx

```diff
@@ -9,8 +9,8 @@ import {
 
 import React, { useEffect, useState } from "react"
 
-import { DailyVoiceClient } from 'react-native-realtime-ai-daily'
-import { TransportState } from 'realtime-ai';
+import { DailyVoiceClient, RNDailyTransport } from '@pipecat-ai/react-native-daily-transport';
```

Comment:
How is the DailyVoiceClient used? Is that not replaced by the RTVIClient?

Reply:
I think that makes sense. Then the transport package is just about providing the transport. That's nice and clean.

Reply:
Done.